### ISSUE TITLE
Detection Score

### ISSUE THREAD

**Issue by chrisbangun** [iitzco/faced#16]
Hi, thanks a lot for open-sourcing faced, this is awesome!

I have two questions:
1. In the `predict` method, do you have any particular reason for not returning the probability as well?

`

    def predict(self, frame, thresh=0.85):
        input_img = cv2.resize(frame, (YOLO_SIZE, YOLO_SIZE)) / 255.
        input_img = np.expand_dims(input_img, axis=0)

        pred = self.sess.run([self.prob, self.x_center, self.y_center, self.w, self.h], feed_dict={self.training: False, self.img: input_img})

        bboxes = self._absolute_bboxes(pred, frame, thresh)
        bboxes = self._correct(frame, bboxes)
        bboxes = self._nonmax_supression(bboxes)

        return bboxes`

2. Related to question 1, is there a convenient way to get the prob/detection score for the predicted image?

Thank you so much.
